__tests__/pages/webhooks/checkout-calculate-taxes.test.ts

```diff
@@ -261,4 +259,181 @@ describe("api/webhooks/checkout-calculate-taxes", () => {
 
     mockJose.mockRestore();
   });
+
+  it("with line that should not have calculated taxes", async () => {
+    const { req, res } = mockRequest({
+      method: "POST",
+      event: "checkout_calculate_taxes",
+      domain: testDomain,
+      signature:
+        "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6..-Y1p0YWNAuX0kOPIhfjoNoyWAkvRl6iMxWQ",
+    });
+
+    const mockJose = jest
+      .spyOn(joseModule, "flattenedVerify")
+      .mockReturnValue(
+        Promise.resolve(
+          {} as unknown as joseModule.FlattenedVerifyResult &
+            joseModule.ResolvedKey
+        )
+      );
```

Comment:
This can be simplified:
```suggestion
      .mockResolvedValue(
          {} as unknown as joseModule.FlattenedVerifyResult &
            joseModule.ResolvedKey
      );
```